vcf-dumper/vcf-dumper-lib/src/main/resources/eva.properties

```diff
@@ -10,3 +10,4 @@ eva.mongo.collections.files=@eva.mongo.collections.files@
 eva.rest.url=http://www.ebi.ac.uk/eva/webservices/rest/
 eva.rest.version=v1
 
+eva.htsget.blocksize=250
```

Comment:
This value is not bad for testing, but for real world operations is too small. We can leave it like this for now.

Reply:
eva.htsget.blocksize is not a test configuration, it's used to determine the block size

Reply:
What I mean is that, if the value for the test and production environments should be different (e.g. 250 for testing, 100000 for production), then they should be placed in 2 different properties files.